Formplayer/testPages/webapps/webapps_basics.py

```diff
@@ -0,0 +1,212 @@
+        self.application_select = (By.XPATH, "//select[@id='report_filter_form_app_id']")
+        self.module_select = (By.XPATH, "//select[@id='report_filter_form_module']")
+        self.form_select = (By.XPATH, "//select[@id='report_filter_form_xmlns']")
+        self.case_type_select = (By.XPATH, "//select[@id='report_filter_case_type']")
+        self.date_input = (By.XPATH, "//input[@id='filter_range']")
+        self.view_form_link = (By.XPATH, "//tbody/tr[1]/td[1]/a[.='View Form']")
+        # self.case_name = (By.XPATH, "//td[div[contains(text(),'abc')]]")
+        self.submit_history_table = (By.XPATH, "//table[@id='report_table_submit_history']/tbody/tr")
+
+        # Case List
+        self.search_input = (By.XPATH, "//input[@id='report_filter_search_query']")
+        self.case_list_table = (By.XPATH, "//table[@id='report_table_case_list']/tbody/tr")
+        self.case_id_block = (By.XPATH, "//th[@title='_id']/following-sibling::td")
+
+        self.click_outside = (By.XPATH, "//label[.//span[text()='Mobile No.']]")
+
+    def application_is_present(self):
+        self.wait_to_click(self.webapps_menu_id)
+        assert self.is_present_and_displayed(self.test_application), "Application is not present in Web Apps"
+        print("App is present")
+
+    def login_as_a_user(self):
+        self.wait_to_click(self.login_as_button)
+        self.wait_to_clear_and_send_keys(self.filter_workers, "appiumtest")
+        self.wait_to_click(self.search_button)
+        self.wait_to_click(self.login_user)
+        self.wait_to_click(self.confirm_login_button)
+
+    def submit_form(self):
+        self.wait_to_click(self.webapps_menu_id)
+        self.login_as_a_user()
+        self.wait_to_click(self.test_application)
+        self.wait_to_click(self.case_list_menu)
+        self.wait_to_click(self.registration_form)
+        self.wait_to_clear_and_send_keys(self.name_question, self.name_input)
+        self.wait_to_click(self.dob_question)
+        self.wait_to_click(self.click_today_date)
+        self.wait_to_click(self.close_date_picker)
+        self.wait_to_clear_and_send_keys(self.mobileno_question, UserData.mobile_number + Keys.TAB)
+        self.js_click(self.submit_form_button)
+        assert self.is_present_and_displayed(self.success_message), ("Form is not submitted!")
+        self.wait_to_click(self.case_list_menu)
+        self.wait_to_click(self.followup_form)
+        self.wait_to_clear_and_send_keys(self.search_case_filter, self.name_input)
+        self.wait_to_click(self.search_button)
+        self.js_click(self.case_name)
+        self.wait_to_click(self.continue_button)
+        self.wait_to_clear_and_send_keys(self.parent_question, self.parent_name_input)
+        self.wait_to_clear_and_send_keys(self.no_of_kids, fetch_random_digit(1,5))
+        self.wait_to_click(self.submit_form_button)
+        assert self.is_present_and_displayed(self.success_message)
+
+    def get_yesterday_tomorrow_dates(self):
```

Comment:
I think the function name needs to be updated here according to the work it does -- (here fetching the current date?)

Reply:
Also the commented code, if not required can be removed